src/pytest_replay/__init__.py

```diff
@@ -84,11 +113,11 @@ def pytest_collection_modifyitems(self, items, config):
             config.hook.pytest_deselected(items=deselected)
             items[:] = remaining
 
-    def append_test_to_script(self, nodeid):
+    def append_test_to_script(self, nodeid, line):
         suffix = "-" + self.xdist_worker_name if self.xdist_worker_name else ""
         fn = os.path.join(self.dir, self.base_script_name + suffix + self.ext)
         with open(fn, "a", encoding="UTF-8") as f:
-            f.write(nodeid + "\n")
+            f.write(line + "\n")
```

Comment:
Ooh we should be flushing after this `write` call.

Reply:
Indeed, I will add it then!